core/wiki/config/ViewTemplateBodyFilters.multids

```diff
@@ -0,0 +1,7 @@
+title: $:/config/ViewTemplateBodyFilters/
+tags: $:/tags/ViewTemplateBodyFilter
+
+system: [prefix[$:/boot/]] [prefix[$:/config/]] [prefix[$:/core/]!field:title[$:/core/readme]!field:title[$:/core/icon]] [prefix[$:/info/]] [prefix[$:/language/]] [prefix[$:/languages/]] [prefix[$:/snippets/]] [prefix[$:/state/]] [prefix[$:/status/]] [prefix[$:/info/]] [prefix[$:/temp/]] +[limit[1]then[$:/core/ui/ViewTemplate/body/code]]
+plugin: [has[plugin-type]then[$:/core/ui/ViewTemplate/body/blank]]
```

Comment:
IMO instead of showing an empty body here, it would be better to show: `$:/core/ui/ViewTemplate/plugin` and remove the tag: `$:/tags/ViewTemplate` that tiddler. 

Since the plugin-type will win here, there is no need in the ViewTemplate to process it again. ... (I'm not sure, if it's clear what I mean)